verboseTree-maven-plugin/src/main/java/org/apache/maven/dependency/graph/OsProperties.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.maven.dependency.graph;
+
+
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+/**
+ * Platform-dependent project properties normalized from ${os.name} and ${os.arch}. Netty uses these to select
+ * system-specific dependencies through the
+ * <a href='https://github.com/trustin/os-maven-plugin'>os-maven-plugin</a>.
+ */
+public class OsProperties
```

Comment:
I'm surprised that this isn't somewhere you could pull it in from, but I can't tell you where to get it from.